content/md/assets/how-to-migrate-from-pam.md

```diff
@@ -0,0 +1,87 @@
+
+## 1-The full-automatic migration
+In this migration, **everything is automated**, you just need to launch one single command and your PAM assets will be migrated in the 4.0 new Asset Manager:
+- One asset family will be created gathering all you assets.
+- All your assets will be migrated and linked to the new asset family.
+- All you existing PAM asset collection product attributes will be linked to the new asset family.
+- And all your links between assets and products / product models will be kept.
+
+You want to try this migration, please follow this step-by-step [full-automatic migration guide](full-automatic-pam-migration.html).
+
+## 2-The family-by-family migration
+In this migration, you can have **several asset families** for your existing PAM assets.
+
+You will need to define manually 2 things:
+- For each asset, its asset family.
+- For each asset collection product attribute, the asset family linked.
+
+Other steps are automated, you just need to launch few commands and your PAM assets will be migrated in the 4.0 new Asset Manager:
+- The asset families will be created automatically in the PIM.
+- All your assets will be migrated and linked to the asset family you defined.
+- All you existing PAM asset collection product attributes will be linked to the asset family.
+- And all your links between assets and products / product models will be kept.
+
+You want to try this migration, please follow the step-by-step [family-by-family migration guide](family-by-family-pam-migration.html).
+
+## 3-The full-manual migration
+In this migration, you will model in the PIM all the asset families you want with their attributes.
+
+# Which migration we advise you?
+First, if you want to go fast in this migration, you can use the [full-automatic migration](full-automatic-pam-migration.html) to test if it fits your needs.
+
+If you have more than 1 asset collection product attribute and these attributes are already split by asset family (for example, an asset collection for Images, an asset collection for Notices...), we recommend you the [family-by-family migration](family-by-family-pam-migration.html).
+
+# What are the limits?
+## No change on assets links to products
+In these 3 migration ways, the assets links to the products are automatically migrated and cannot be modified.
+
+It means that you can't modify the asset links to products and the number of asset collection product attribute.
+
+**Why?** Because links to products are vital and we would like to keep the existing asset product links and do not impact your sales channels with this migration.
+
+ <!--TODO
+If you really want to change your asset collection product attribute, you need to do it manually by exporting the products, splitting...
+But be careful...-->
+
+# What about my connectors?
+**Your connectors that previously import or export your DAM assets need to be updated for the 4.0 new Asset Manager**.
```

Comment:
we could say that if they used the csv import, they can use the tool we are building